config/crd/bases/toolchain.dev.openshift.com_spaceprovisionerconfigs.yaml

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
I'm not sure what's going on but there is a bunch of unrelated changes in the PR which has been already merged into the master branch. Not sure why GitHub still showing them here...
Like this CRD for example.